### PR TITLE
added sold training tickets to conference admin stats

### PR DESCRIPTION
**Pull request by artcz**

### CODE REVIEW

File: tests/common_tools.py [hunk 1]
```diff
@@ -124,10 +124,10 @@ def sequence_equals(sequence1, sequence2):
     return True
 
 
-def make_user():
+def make_user(**kwargs):
     user = auth_factories.UserFactory(
         email='joedoe@example.com', is_active=True,
-        is_staff=True  # TEMPORARY FOR PDF DEBUG TESTS
+        **kwargs
```
Comment (matrixise) [inline tests/common_tools.py]: in the csv export for the accountant, we check if the user is a staff member. I think we will fix some tests ;-)

File: tests/test_conference_stats_in_admin.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+def test_if_stats_are_accessible_in_admin(admin_client):
+    """
+    Mainly tests if we don't get a random 500 due to an obvious bug in a lower
+    level code.
+    """
+    url = "/admin/conference/conference/ep2018/stats/"
+    HTTP_OK_200 = 200
+
+    response = admin_client.get(url)
+    assert response.status_code == HTTP_OK_200
```
Comment (matrixise) [inline tests/test_conference_stats_in_admin.py:33]: add the test when the user is not an admin ;-)

**Review by matrixise — APPROVED** [review]

File: tests/test_conference_stats_in_admin.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+HTTP_LOGIN_REQUIRED_REDIRECT_302 = 302
+
+
+@mark.django_db
+def test_if_stats_are_not_accessible_to_regular_users(client):
+    """
+    Mainly tests if we don't get a random 500 due to an obvious bug in a lower
+    level code.
+    """
+    make_user(is_staff=False)
```
Comment (matrixise) [inline tests/test_conference_stats_in_admin.py]: I prefer

```python
user = make_user(is_staff=False, password='joedoe')
client.login(email=user.email, password='joedoe')
```
and you?

**Review by matrixise — CHANGES_REQUESTED** [review]

File: tests/test_conference_stats_in_admin.py (hunk 3 above)
Comment (artcz) [reply tests/test_conference_stats_in_admin.py]: fixed

**Comment by umgelurgel** [comment]
This seems to have been implemented separately, it's available as of ep2019.

**Pull request closed without merging**